### PR TITLE
Convert pointer language to symmetric address language in memory management routines

### PR DESCRIPTION
**Pull request by agrippa**

### CODE REVIEW

**Comment by agrippa** [comment]
@manjugv @minsii @swpoole @jamesaross @tonycurtis @jdinan please take a look

**Comment by agrippa** [comment]
In response to issue https://github.com/openshmem-org/specification/issues/351

File: content/shmem_malloc.tex [hunk 1]
```diff
@@ -82,12 +82,12 @@
...
+    The \FUNC{shmem\_malloc} routine returns the symmetric address of the allocated space;
     otherwise, it returns a null pointer.
     
     The \FUNC{shmem\_free} routine returns no value.
     
-    The \FUNC{shmem\_realloc} routine returns a pointer to the allocated space
+    The \FUNC{shmem\_realloc} routine returns the symmetric address of the allocated space
     (which may have moved); otherwise, all \acp{PE} return a null pointer.
     
     The \FUNC{shmem\_align} routine returns an aligned pointer whose value is a
```
Comment (minsii) [inline content/shmem_malloc.tex]: Do we want to replace `aligned pointer` with `aligned symmetric address`?

**Review by jdinan — APPROVED** [review]
💯

**Review by manjugv — APPROVED** [review]

**Review by minsii — APPROVED** [review]

**Review by tonycurtis — APPROVED** [review]

**Comment by swpoole** [comment]
Approved.

**Comment by manjugv** [comment]
@agrippa Can I merge this ?

**Comment by agrippa** [comment]
@manjugv yep, this is ready to go.

**Comment by jamesaross** [comment]
👍

**Pull request merged**